### PR TITLE
change controllerDispatcher to setTimeOut update

### PR DESCRIPTION
**Pull request by druiz17**
change controllerDispatcher to setTimeOut update because script.update to to long to start

ticket - https://highfidelity.fogbugz.com/f/cases/edit/9352/Move-controllerDispatcher-from-Script-update-to-Script-setTimeOut

### CODE REVIEW

**Comment by druiz17** [comment]
Test plan
- this PR should have no function to controller modules.
- run the test plans for grabbing, tablet interaction just to clarify that the scripts work as before.

**Comment by Nex-Pro** [comment]
@druiz17 More detail/links in the test notes please.

**Comment by druiz17** [comment]
@Nex-Pro added the link of test you should run.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/5014/HighFidelity-Beta-PR11833-c82798673fb342cecf0db20a7edf1f599156d4a8.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/5014/HighFidelity-Beta-PR11833-c82798673fb342cecf0db20a7edf1f599156d4a8.dmg

**Comment by Nex-Pro** [comment]
Testing

**Comment by Nex-Pro** [comment]
@druiz17 All 11 test cases plus tablet interactions ?

**Comment by druiz17** [comment]
Yes. I just want to make sure they all work. There is not functional changes, so they should behave the same way as before.

**Comment by Nex-Pro** [comment]
Multiple crashes during and after startup. Splats submitted.

debug.
Unhandled exception at 0x00007FFD697084CC (Qt5Script.dll) in interface.exe: 0xC0000005: Access violation writing location 0x000001EC53F54000. occurred

**Comment by Nex-Pro** [comment]
@druiz17 ^ Ouch

**Comment by Nex-Pro** [comment]
### FAIL

**Comment by druiz17** [comment]
@Nex-Pro I fixed the crash

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/5040/HighFidelity-Beta-PR11833-cabe3819a333e68ad97be7ae1eb0c5e93b67aa79.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/5040/HighFidelity-Beta-PR11833-cabe3819a333e68ad97be7ae1eb0c5e93b67aa79.dmg

File: scripts/system/controllers/controllerDispatcher.js [hunk 1]
```diff
@@ -367,6 +367,7 @@ Script.include("/~/system/libraries/controllerDispatcherUtils.js");
             if (PROFILE) {
                 Script.endProfileRange("dispatch.run");
             }
+            Script.setTimeout(_this.update, 60);
```
Comment (ZappoMan) [inline scripts/system/controllers/controllerDispatcher.js]: is 60 right? won't that be like 16hz? `setTimeout` takes milliseconds as input

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/5107/HighFidelity-Beta-PR11833-9622fef149ea365c37aac4b293715a2987701c02.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/5107/HighFidelity-Beta-PR11833-9622fef149ea365c37aac4b293715a2987701c02.dmg

**Comment by YuppyPlays** [comment]
Testing
[w/ Cain]

**Comment by YuppyPlays** [comment]
Testing complete:

Had a few **fails** shown in the TestRail Results located here: https://highfidelity.testrail.net/index.php?/runs/view/1160&group_by=cases:section_id&group_order=asc

We experienced an initial delay as we were having some really strange connectivity issues, but we figured this was non-PR related.

**Comment by themelissabrown** [comment]
@druiz17 @YuppyPlays are those fails in test rail always fails in master? Or specific to this PR?

**Comment by YuppyPlays** [comment]
@themelissabrown 

Regarding the 3 fails and relating to master:
T17071 - The laser issue I have seen in previous Master builds but I believe it was fixed - but I have not checked latest, will have a look with Cain to confirm later today when he's available.

T17073 - Issue with marketplace giving public key errors I can confirm is not in latest Master, works as normal.

T17075 - This one was a fail due to not being able to rez any hand attachments due to same error as before (Public key error) not present in Master.

(Master 7474 is the version I currently have to test against)

**Comment by druiz17** [comment]
This PR does not change any grabbing functionality. It just allows the controllerScripts to run right away instead of wait for the physics to run. Lasers, stylus and grabbing behavior should still work, which is what I am trying to confirm. Also the test fails mentioned above have nothing to do with the grabbing behavior. They seem to be about rez rights and wallet issue which tickets should be created for. I also just updated this branch with latest mater.

**Comment by druiz17** [comment]
@YuppyPlays Can I get this QA again and also read my comment above.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/5290/HighFidelity-Beta-PR11833-6fd8fb1e5b78052ee494fb4509a9e8c75a88ead8.dmg

**Comment by marko8904** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/5294/HighFidelity-Beta-PR11833-6fd8fb1e5b78052ee494fb4509a9e8c75a88ead8.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/5294/HighFidelity-Beta-PR11833-6fd8fb1e5b78052ee494fb4509a9e8c75a88ead8.dmg

**Comment by YuppyPlays** [comment]
@druiz17 
My sincerest apologies for the delay in response, Cain is currently flying back home and should be available at some time tomorrow, we can both do this together tomorrow evening if this is okay :)

**Comment by YuppyPlays** [comment]
Starting this shortly, just getting our setups ready.

**Comment by YuppyPlays** [comment]
Testing.

**Comment by YuppyPlays** [comment]
Testing Failed.

Had a crucial issue on both our Interfaces where we could not interact with tablet using stylus and teleportation did not seem to work either. (Restarting did not fix this either)

Have not experienced this in any other PR or Master.

**Tested w/ Cain**

**Comment by marko8904** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/5512/HighFidelity-Beta-PR11833-0286f4646a53b95cf52291bd1ce847d0d05b293c.dmg

**Comment by druiz17** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/5521/HighFidelity-Beta-PR11833-0286f4646a53b95cf52291bd1ce847d0d05b293c.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/5521/HighFidelity-Beta-PR11833-0286f4646a53b95cf52291bd1ce847d0d05b293c.exe

**Comment by Nex-Pro** [comment]
Testing shortly by request of @druiz17

**Comment by Nex-Pro** [comment]
Testing with two users. 2x Vive.

**Comment by Nex-Pro** [comment]
### FAIL

failed 6 of 11 testrail cases.

https://highfidelity.testrail.net/index.php?/runs/view/1437&group_by=cases:section_id&group_order=asc

Two users local sandbox two vives.

**Comment by tapalisa** [comment]
The QA failure yuppy experienced seems to have gone away. Would like to merge this

**Pull request merged**